remove hardcoded paths from scripts
authors: Denis Kristak (INUITS), Kenneth Hoste (HPC-UGent)
Updated to 24.10.07: J. Sassmannshausen (Imperial College London/UK)
diff --git a/r-inla-Version_24.10.07.orig/rinla/inst/bin/linux/64bit/fmesher.run b/r-inla-Version_24.10.07/rinla/inst/bin/linux/64bit/fmesher.run
index 5a273e9..3195d02 100755
--- a/r-inla-Version_24.10.07.orig/rinla/inst/bin/linux/64bit/fmesher.run
+++ b/r-inla-Version_24.10.07/rinla/inst/bin/linux/64bit/fmesher.run
@@ -4,18 +4,6 @@ tmp=$(basename "$cmd")
 prog=${tmp%%.run}
 DIR=$(dirname "$cmd")
 
-D=""
-for d in {,/usr}/lib64 /usr/lib64/R/lib {,/usr}/lib/x86_64-linux-gnu {,/usr}/lib; do
-    if [ -d "$d" ]; then
-	if [ -z "$D" ]; then
-	    D="$d"
-	else
-	    D="$D:$d"
-	fi
-    fi
-done
-export LD_LIBRARY_PATH="$DIR/first:$D:$DIR:$LD_LIBRARY_PATH"
-
 if [ "${INLA_DEBUG}XX" != "XX" ]; then
     echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH"
     ldd -r "$DIR/$prog"
diff --git a/r-inla-Version_24.10.07.orig/rinla/inst/bin/linux/64bit/inla.run b/r-inla-Version_24.10.07/rinla/inst/bin/linux/64bit/inla.run
index 49a6424..555d007 100755
--- a/r-inla-Version_24.10.07.orig/rinla/inst/bin/linux/64bit/inla.run
+++ b/r-inla-Version_24.10.07/rinla/inst/bin/linux/64bit/inla.run
@@ -13,17 +13,6 @@ if [ ! -z ${R_HOME+x} ]; then
     fi
 fi
 
-D=""
-for d in {,/usr}/lib64 /usr/lib64/R/lib {,/usr}/lib/x86_64-linux-gnu {,/usr}/lib; do
-    if [ -d "$d" ]; then
-	if [ -z "$D" ]; then
-	    D="$d"
-	else
-	    D="$D:$d"
-	fi
-    fi
-done
-
 if [ -n "${INLA_NATIVE_LD_LIBRARY_PATH}" ]; then
     ## so we can revert back to old behaviour
     export LD_LIBRARY_PATH="$DIR/first:$D:$DIR:$LD_LIBRARY_PATH"
